readme.md

```diff
@@ -11,9 +11,21 @@
     </a>
 </p>
 
+## Installation
+
+Install with [Fisher](https://github.com/jorgebucaran/fisher):
+
+```console
+fisher install rose-pine/fish
+```
+
+Install manually:
+
+Download the variant of your choice from the `themes` and place it inside the `~/.config/fish/themes`
+
 ## Usage
 
-1. Move one of `rose-pine{-dawn,-moon}.fish` to `~/.config/fish/conf.d`
+1. Add `fish_config theme choose "Rose Pine {Dawn, Moon}` to `~/.config/fish/config.fish`
```

Comment:
I know this grammar is used in other themes but technically it should be Rosé instead of Rose.

Sorry for multiple edits but I just remembered the build file names will spit out snake-case names, e.g. "rose-pine.fish" instead. If this is an issue, I am happy to update the build script to accommodate this but if snake-case works I think we should stick to that.

Reply:
Also, looks like a quote is missing:

```suggestion
1. Add `fish_config theme choose "Rose Pine {Dawn, Moon}"` to `~/.config/fish/config.fish`
```

Reply:
> I know this grammar is used in other themes but technically it should be Rosé instead of Rose. I was going to suggest using "rose-pine" or "rose_pine" but no other themes seem to do that :/
> 
> Even though "Rosé" could be harder for some to type vs "Rose", perhaps "Ros<tab>" would autocomplete?

I'm pretty sure "Ros<tab" would autocomplete just fine! I'm going to change the names. :

Reply:
I'll take the freedom to go for `Rosé Pine {Dawn, Moon}.theme`, but please let me know if that doesn't look good. I've just tested changing the "Rose" in the names for "Rosé" and the autocomplete works just fine 👌